src/main/fc/fc_rc.c

```diff
@@ -259,9 +265,14 @@ FAST_CODE uint8_t processRcInterpolation(void)
 }
 
 #ifdef USE_RC_SMOOTHING_FILTER
-uint8_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline)
+uint16_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline, bool pt1)
 {
-    return lrintf(filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS));
```

Comment:
This formula becomes misleading if you assume `filterBaseline` is frequency and `avgRxFrameRate` is also frequency:
`[1 / s] / ([1 / s] / s) = [1 / s] / [1 / s2] = [s]`
While really it is:
`[1 / s] / ([s] / [s]) = [1 / s]`

That's why I'm asking to rename variables or refactor this by making `filterBaseline` dimensionless.